.github/workflows/test.yml

```diff
@@ -16,9 +16,13 @@ jobs:
       - run: |
           npm run all
   test: # make sure the action works on a clean machine without building
+    permissions:
+      id-token: write
+      contents: read
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
       - uses: ./
         with:
-          milliseconds: 1000
+          awsRoleToAssume: ${{ secrets.GU_RIFF_RAFF_ROLE_ARN }}
+      - run: aws s3 cp README.md s3://riffraff-artifact/actions-assume-aws-role/ci/action.yml
```

Comment:
Uploading random files to Riff-Raff's bucket is totally fine, right?!

Reply:
Should the bucket name be stored in a repo secret?

Reply:
> Should the bucket name be stored in a repo secret?

Yes, I think that would be preferable.